keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/ApplicationLoadBalancerModel.kt

```diff
@@ -68,7 +68,7 @@ data class ApplicationLoadBalancerModel(
     val port: Int,
     val healthCheckEnabled: Boolean,
     val healthCheckTimeoutSeconds: Int,
-    val healthCheckPort: Int,
+    val healthCheckPort: String, // quoted number (e.g., "8080") or "traffic-port"
```

Comment:
is `traffic-port` the only string that can appear there?

Reply:
I think so, based on [similar logic inside of clouddriver](https://github.com/spinnaker/clouddriver/blob/7c861e3b1c47db1e103df37dc410930b36aa61fb/clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/loadbalancer/UpsertAmazonLoadBalancerV2AtomicOperation.groovy#L160).